### DISCUSSION TITLE
student pack

### DISCUSSION THREAD

**Discussion by dungz3us** [community/community#49215] [GitHub Education]
### Select Topic Area

Product Feedback

### Body

i reapply mu student developer pack with my studentID but Github require more. 
Why my studentID photo is not enough

**Comment by jge162** [comment]
Sorry to hear you are having issues. Please refer back to the documentation as it gives full guide and what you need. Here is the [link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)

**Reply by dungz3us** [reply]
I've already seen it. I've send a few time but it just rejected
